ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -299,6 +304,7 @@
   private final String replicationPolicy;
   private final boolean ttlRequired;
   private final short parentAccountId;
+  private final List<CloudReplicationConfig> cloudConfigs;
```

Comment:
Would it simplify the code a little if you used `Collections.emptyList()` instead of null as the default?

Reply:
Have to see how this works with JSON's `opt`